Makefile

```diff
@@ -287,6 +295,8 @@ build/$(SPEC): $(SPEC)
 	$(CPP) $(CPPFLAGS) $< > $@
 
 build/ldscript.txt: build/$(SPEC)
+	python3 tools/assemble_sound.py assets/soundfonts build/assets build/include assets/samples --build-bank --match ocarina
+	python3 tools/assemble_sequences.py src/audio assets/sequences build/include build
```

Comment:
Yeah, the targets were entirely chosen just to make it _build_, not because it was a good place for them. 😅 I can try structuring a couple targets like that, let me give it a shot.